src/main/views/address/postcodeEnter.njk

```diff
@@ -12,14 +12,21 @@
       }
     }) }}
 
-    {{ govukButton({
-      attributes: {
-        id: "findAddressButton"
-      },
-      text: findAddress,
-      type: "submit",
-      preventDoubleClick: true
-    }) }}
+    <div class="govuk-grid-row">
```

Comment:
There is a CSS class for grouping two buttons together, side by side - "govuk-button-group" - should improve the alignment.